static/app/views/admin/adminQuotas.tsx

```diff
@@ -1,89 +1,88 @@
 import {Fragment} from 'react';
 
-import DeprecatedAsyncComponent from 'sentry/components/deprecatedAsyncComponent';
 import TextField from 'sentry/components/forms/fields/textField';
 import InternalStatChart from 'sentry/components/internalStatChart';
+import LoadingError from 'sentry/components/loadingError';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
 import Panel from 'sentry/components/panels/panel';
 import PanelBody from 'sentry/components/panels/panelBody';
 import PanelHeader from 'sentry/components/panels/panelHeader';
 import {t} from 'sentry/locale';
+import {useApiQuery} from 'sentry/utils/queryClient';
 
 type Config = {
   backend: string;
   options: Record<string, string>;
 };
 
-type State = DeprecatedAsyncComponent['state'] & {
-  config: Config;
-  resolution: string;
-  since: number;
-};
+export default function AdminQuotas() {
+  const {
+    data: config,
+    isPending,
+    isError,
+  } = useApiQuery<Config>(['/internal/quotas/'], {
+    staleTime: 0,
+  });
+  const [since] = useState(() => new Date().getTime() / 1000 - 3600 * 24 * 7)
+  const resolution = '1h';
```

Comment:
@scttcper same for resolution?

```suggestion
  const resolution = useState(() => '1h');
```

Reply:
when it's a string that doesn't change you're good. the date as a number can change every rerender